src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua

```diff
@@ -837,9 +848,22 @@ function Adjustable.Container:new(cons,container)
     shrink_title(me)
     me.lockStyle = me.lockStyle or "standard"
     me.noLimit = me.noLimit or false
-    me.raiseOnClick = me.raiseOnClick or true
+    if not(me.raiseOnClick == false) then
+        me.raiseOnClick = true
+    end
     -- save a list of all containers in this table
-    Adjustable.Container.all[#Adjustable.Container.all+1] = me
+    Adjustable.Container.all[#Adjustable.Container.all + 1] = me
+
+    -- Loads on creation (by Name) if autoLoad is not false
+    if not(me.autoLoad == false) then
+        me.autoLoad = true
+        me:load()
+    end
+    -- Saves on Exit if autoSave is not false
+    if not(me.autoSave == false) then
```

Comment:
Remember to handle someone re-running the script with different arguments, the state needs to be changed

Reply:
After you saying that I noticed that https://github.com/Mudlet/Mudlet/pull/3613 prevents everything from reloading states if someone reruns a script :facepalm: :sob:

Reply:
Good catch. It's something we have to be careful about in all Geyser constructors, that warrants a comment in code.

Reply:
have to find another solution for https://github.com/Mudlet/Mudlet/pull/3613 then...